doc/source/multiscene.rst

```diff
@@ -231,3 +231,46 @@ multiple Scenes use:
     >>> mscn = MultiScene.from_files(glob('/data/abi/day_1/*C0[12]*.nc'), reader='abi_l1b')
     >>> mscn.load(['C01', 'C02'])
     >>> mscn.save_datasets(base_dir='/path/for/output')
+
+Combining multiple readers
+--------------------------
+
+Since Satpy 0.23, it is possible to automatically
+combine multiple readers into a single MultiScene using the
+:meth:`~satpy.multiscene.MultiScene.from_files` constructor.  It is
+no longer necessary for the user to create the :class:`~satpy.scene.Scene`
+objects themselves.  For example, you can combine Advanced Baseline
+Imager (ABI) and Global Lightning Mapper (GLM) measurements.
+Constructing a multi-reader MultiScene requires more parameters
+than a single-reader MultiScene, because Satpy can poorly guess how
+to group files belonging to different instruments.  For an example
+creating a video with lightning superimposed on ABI channel 14 (11.2
+micrometre), using the built-in composite ``C14_flash_extent_density``,
```

Comment:
Controversial: `11.2um` (or the actual mu character) or `11.2 micrometer` since we are technically US English.

Reply:
Changed to 11.2 µm.